### PR TITLE
Gradle updates

### PR DESCRIPTION
**Pull request by CodeCombiner**

### CODE REVIEW

File: android/GameCircleSDKLibrary/GameCircleSDK/build.gradle [hunk 1]
```diff
@@ -0,0 +1,29 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 27
+    buildToolsVersion '27.0.3'
+
+    defaultConfig {
+        minSdkVersion 19
+        targetSdkVersion 27
+        multiDexEnabled true
```
Comment (rampr) [inline android/GameCircleSDKLibrary/GameCircleSDK/build.gradle]: can we do away with multiDex

File: android/config.json [hunk 2]
```diff
@@ -2,10 +2,8 @@
     "copyFiles": [
         "GameCirclePlugin.java"
     ],
-    "libraries": [
-		"GameCircleSDK"
-    ],
     "jars": [
+    "GameCircleSDKLibrary/GameCircleSDK/build/outputs/aar/GameCircleSDK-debug.aar"
```
Comment (rampr) [inline android/config.json]: indentation

File: android/GameCircleSDKLibrary/proguard-project.txt [hunk 3]
```diff
@@ -18,3 +18,12 @@
 #-keepclassmembers class fqcn.of.javascript.interface.for.webview {
```
Comment (rampr) [inline android/GameCircleSDKLibrary/proguard-project.txt]: the permissions have changed for 644 to 755. Is that intentional ?

File: android/GameCircleSDKLibrary/build.gradle [hunk 4]
```diff
@@ -0,0 +1,17 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+    repositories {
+        jcenter()
+        google()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.0.0'
```
Comment (rampr) [inline android/GameCircleSDKLibrary/build.gradle]: gradle version

File: android/GameCircleSDKLibrary/proguard-project.txt (hunk 3 above)
Comment (CodeCombiner) [reply android/GameCircleSDKLibrary/proguard-project.txt]: I can not remember if I had any issue with this. Is that important?

File: android/GameCircleSDKLibrary/GameCircleSDK/build.gradle (hunk 1 above)
Comment (CodeCombiner) [reply android/GameCircleSDKLibrary/GameCircleSDK/build.gradle]: Removed

File: android/GameCircleSDKLibrary/proguard-project.txt (hunk 3 above)
Comment (CodeCombiner) [reply android/GameCircleSDKLibrary/proguard-project.txt]: Uncommented proguard parameter

Comment (rampr) [reply android/GameCircleSDKLibrary/proguard-project.txt]: I mean the permissions of the file have changed from 644 to 755
rw-w-w to rwx-rw-rw.
I think this is fine

**Review by rampr — APPROVED** [review]

**Pull request merged**